issues.php

```diff
@@ -11,15 +24,15 @@
 );
 
 function wikiGithubIssues() {
-    global $wgParser;    
+    global $wgParser;
     $wgParser->setHook("githubissues", "embedGithubIssues");
 }
 
 function embedGithubIssues($input, $args) {
 	global $wgParser;
```

Comment:
You'll need to add `$wgGithubIssuesShowComments` to the global line here in order for it to be accessible inside this function.